obsreport/obsreport_receiver.go

```diff
@@ -63,7 +83,74 @@ func NewReceiver(cfg ReceiverSettings) *Receiver {
 			tag.Upsert(obsmetrics.TagKeyTransport, cfg.Transport, tag.WithTTL(tag.TTLNoPropagation)),
 		},
 		tracer: cfg.ReceiverCreateSettings.TracerProvider.Tracer(cfg.ReceiverID.String()),
+		meter:  cfg.ReceiverCreateSettings.MeterProvider.Meter(cfg.ReceiverID.String()),
```

Comment:
Not sure if this is the right name for the meter. Should we have a meter per factory type of per factory component instance?

Reply:
Do you mean to use `receiver/otlp` for the `otlp` receiver's meter name instead of just `otlp`, to differentiate components that have the same name? 

If that is what you meant, I think that seems better. Should tracer name be like that as well?

Reply:
Nope, I mean to use `cfg.ReceiverID.Type()`, so if we have multiple instances of the `otlp` receiver we have same scope for this, and the instance  name is not in the scope name (probably should by an attribute).

Reply:
I don't think `otlp` is an appropriate scope name.  Shouldn't the scope here be `go.opentelemetry.io/collector/obsreport` with attributes for `componentType=receiver` and `componentID=otlp` (or `componentID=otlp/foo` if a name exists on the receiver ID)?

Reply:
That is also a possibility, but you are still suggesting the same thing, that component name should not be part of the scope (which was before).

I think I like scope.name=`go.opentelemetry.io/collector/obsreport/receiver` and have a `component.id=otlp` (or `component.id=otlp/foo`) since we use `.` mostly as attribute name separator in otel (see resource attributes for example).

Reply:
By the way, we will encounter some warns/missing metrics due to some otel-go bugs at the moment. 

Because the go sdk doesn't return a valid instrument for "duplicate" metric creation results in invalid instruments in otel-go ([tracking issue](https://github.com/open-telemetry/opentelemetry-go/issues/3229
)), only the first receiver would have valid instruments and subsequent calls to the `obsreport.NewReceiver()` will have noop metrics.

I would suggest us to just add the `component.id` (`cfg.ReceiverID.String()`) as suffix for now, adding a TODO with the tracking issue and remove that once that gets fixed. Since the prometheus exporter doesn't use the scope for nothing yet, it wouldn't change anything.

Reply:
I would not do that, since that will be a breaking change. But fix was release today.

Reply:
Right, upgrading to the new released version fixed this.